apps/desktop/src/main/terminal-host/session.ts

```
@@ -128,6 +135,37 @@ type SpawnProcess = (
 	options: Parameters<typeof spawn>[2],
 ) => ChildProcess;
 
+function summarizeTerminalData(data: string): Record<string, unknown> {
+	return {
+		bytes: Buffer.byteLength(data, "utf8"),
+		chars: data.length,
+		escapes: (data.match(/\x1b/g) || []).length,
```

Biome (2.4.11)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)